### PR TITLE
V8: Fix "infinityMode" typo

### LINKED ISSUE #6058: V8: infiniteMode or infinityMode - typo?
Whilst developing a custom property-editor, I'd noticed in the angularjs controller `model` there are properties for both **`infiniteMode`** and **`infinityMode`**.  I can't help but think that the latter is a typo.

I've searched over the latest (v8.1.1) source code and found the following references...

- https://github.com/umbraco/Umbraco-CMS/blob/release-8.1.1/src/Umbraco.Web.UI.Client/src/views/components/editor/umb-editors.html#L8
- https://github.com/umbraco/Umbraco-CMS/blob/release-8.1.1/src/Umbraco.Web.UI.Client/src/common/directives/components/editor/umbeditors.directive.js#L20
- https://github.com/umbraco/Umbraco-CMS/blob/release-8.1.1/src/Umbraco.Web.UI.Client/src/less/components/editor/umb-editor.less#L20-L27
- https://github.com/umbraco/Umbraco-CMS/blob/release-8.1.1/src/Umbraco.Web.UI.Client/src/less/components/editor/subheader/umb-editor-sub-header.less#L42-L44

Is this a typo? Should we correct it? _(I'm not sure what this means in terms of a breaking-change)_

### PR DESCRIPTION
**Pull request by kjac**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes #6058

### Description

This PR fixes the "infinityMode" typo originating from the `umbEditors` directive.

As it turns out, `umbEditors` attempts to dictate infinite mode - only it kinda fails as it's setting `infinityMode`, not `infiniteMode` which is used throughout the app. But `umbEditors` really shouldn't be messing with infinite mode. This is determined and handled by `editorService` [here](https://github.com/umbraco/Umbraco-CMS/blob/a398881fa1e7eebda4ad74d505a592d4178bfa64/src/Umbraco.Web.UI.Client/src/common/services/editor.service.js#L265).

Bottom line: We can safely remove the `infinityMode` assignment from `umbEditors` and rely on the `infiniteMode` assignment from `editorService`. That's what this PR does 😄 

#### Testing this PR

Browse around the backoffice and open things in infinite editing (both single level and multilevel infinite editing) and make sure the infinite editing dialogs look and behave as expected.

### CODE REVIEW

**Comment by leekelleher** [comment]
@kjac Thanks for picking up on this one!

Just looking over your changes, think there's also this one in "umb-editor-sub-header.less"...
https://github.com/umbraco/Umbraco-CMS/blob/release-8.1.1/src/Umbraco.Web.UI.Client/src/less/components/editor/subheader/umb-editor-sub-header.less#L42-L44

**Comment by kjac** [comment]
@leekelleher thanks for reviewing 👍  y

Yeah I was puzzled too that I could only find three occurrences of "infinityMode" when clearly you had linked to four. Turns out umb-editor-sub-header.less has changed since 8.1.1 - the [most recent version](https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Web.UI.Client/src/less/components/editor/subheader/umb-editor-sub-header.less) does not contain that mention of "infinityMode".

**Comment by leekelleher** [comment]
@kjac Ah, cool. 😎 

Found the change here: https://github.com/umbraco/Umbraco-CMS/commit/d17022d87dcdb6a6867be6caf2da5f7f36cec47b#diff-c4783cfffd16db743bb788803974751fL42

Although, I'm a bit confused about the dates of that commit. The commit itself was 4th June, (I assume, in a different branch), and the merge commit (to "v8/dev" branch) was 9th July ... and v8.1.1 was tagged on 24th July. 🤔  ... and I'm not sure how the "v8/8.1" branch comes into play with this. Bit confusing. 🤷‍♂

**Comment by leekelleher** [comment]
I've got it now, "v8/dev" is for next minor (e.g. 8.2.0) and "v8/8.1" is the further patch releases for that minor.  Which I assume with merge in with "v8/dev" after the next patch release (8.1.2). All good.

**Comment by nul800sebastiaan** [comment]
Correct @leekelleher - we branched `v8/8.1` off of `v8/dev` - `v8/dev` is for next minor and changes we make in `v8/8.1` always get merged up to `v8/dev`. 

Thanks for the investigation and what a relief this is unused @kjac ! 🎉
